counterpoll/main.py

```diff
@@ -222,6 +261,8 @@ def show():
         data.append(["PORT_STAT", port_info.get("POLL_INTERVAL", DEFLT_1_SEC), port_info.get("FLEX_COUNTER_STATUS", DISABLE)])
     if port_drop_info:
         data.append([PORT_BUFFER_DROP, port_drop_info.get("POLL_INTERVAL", DEFLT_60_SEC), port_drop_info.get("FLEX_COUNTER_STATUS", DISABLE)])
+    if pg_drop_info:
+        data.append(['PG_DROP_STAT', port_drop_info.get("POLL_INTERVAL", DEFLT_10_SEC), pg_drop_info.get("FLEX_COUNTER_STATUS", DISABLE)])
```

Comment:
@ayurkiv-nvda please move it after `PG_WATERMARK_STAT`

Reply:
Done